test-files/algorithms/wayjoiner/WayJoinerAdvancedConflateExpected.osm

```diff
@@ -150,23 +150,23 @@
         <nd ref="-34"/>
         <nd ref="-33"/>
         <nd ref="-30"/>
+        <tag k="hoot:status" v="3"/>
         <tag k="highway" v="road"/>
         <tag k="note" v="2"/>
-        <tag k="hoot:status" v="3"/>
-        <tag k="error:circular" v="15"/>
         <tag k="hoot:id" v="-192"/>
+        <tag k="error:circular" v="15"/>
     </way>
     <way visible="true" id="-185" timestamp="1970-01-01T00:00:00Z" version="1">
         <nd ref="-32"/>
         <nd ref="-2"/>
         <nd ref="-1"/>
+        <tag k="hoot:status" v="3"/>
         <tag k="highway" v="road"/>
         <tag k="note" v="1"/>
-        <tag k="hoot:status" v="3"/>
-        <tag k="error:circular" v="15"/>
         <tag k="hoot:id" v="-185"/>
+        <tag k="error:circular" v="15"/>
     </way>
-    <way visible="true" id="-178" timestamp="1970-01-01T00:00:00Z" version="1">
+    <way visible="true" id="-172" timestamp="1970-01-01T00:00:00Z" version="1">
```

Comment:
There are quite a few of these changes where the ID that is kept is now larger if it is negative or smaller if it is positive.  See explanation above in the `WayJoiner` class.